libs/libumqtt/Makefile

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright (C) 2014-2017 OpenWrt.org
```

Comment:
No, either use your own name here, or drop this. you can't assign copyright to openwrt.org like this.

Reply:
Done.